### DISCUSSION TITLE
Install via git vs postgresql-15-partman package

### DISCUSSION THREAD

**Discussion by themactech** [pgpartman/pg_partman#526] [Q&A]
I was looking through the packages offered by the repo from the postgresql community, and saw there is a package called postgresql-15-partman in there.  Does that mean this can completely replace the "build from git" approach with a simple apt install (easier to manage in ansible as well)?

**Comment by keithf4** [comment]
Yes it does! I'll see about adding a note for that in the readme for my next release
